src/domains/ecommerce-common/00_alerts.tf

```diff
@@ -254,4 +254,49 @@ AzureDiagnostics
     operator  = "GreaterThanOrEqual"
     threshold = 2
   }
+}
+
+# eCommerce user stats service availability
+resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_user_stats_service_availability_alert" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "ecommerce-user-stats-service-availability-alert"
+  resource_group_name = azurerm_resource_group.rg_ecommerce_alerts[0].name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id]
+    email_subject          = "[eCommerce] User stats service availability less that 99%"
+    custom_webhook_payload = "{}"
+  }
+  data_source_id = data.azurerm_api_management.apim.id
+  description    = "eCommerce User stats service availability less than 98% in the last 30 minutes detected"
+  enabled        = true
+  query = (<<-QUERY
+let thresholdTrafficMin = 200;
+let thresholdTrafficLinear = 500;
+let lowTrafficAvailability = 94;
+let highTrafficAvailability = 98;
+let thresholdDelta = thresholdTrafficLinear - thresholdTrafficMin;
+let availabilityDelta = highTrafficAvailability - lowTrafficAvailability;
+AzureDiagnostics
+| where url_s startswith 'https://api.dev.platform.pagopa.it/ecommerce/user-stats-service/v1'
+| summarize
+    Total=count(),
+    Success=countif(responseCode_d < 500 and DurationMs < 500)
+    by Time = bin(TimeGenerated, 15m)
+| extend trafficUp = Total-thresholdTrafficMin
+| extend deltaRatio = todouble(todouble(trafficUp)/todouble(thresholdDelta))
+| extend expectedAvailability = iff(Total >= thresholdTrafficLinear, toreal(highTrafficAvailability), iff(Total <= thresholdTrafficMin, toreal(lowTrafficAvailability), (deltaRatio*(availabilityDelta))+lowTrafficAvailability)) 
+| extend Availability=((Success * 1.0) / Total) * 100
+| where Availability < expectedAvailability
```

Comment:
Maybe this query could be simpler than this. This kind of query is useful for those apis that is used in user experience traffic. The api of `lastIUsageUpdate` instead is used for intra service communication and could follow a _traditional_ query like this one:
`
AzureDiagnostics 
| where url_s matches regex "https://api.platform.pagopa.it/ecommerce/user-stats-service/v1"
| where method_s == "POST"
| where responseCode_d != 200 or DurationMs > 2000
| project TimeGenerated, responseCode_d, DurationMs
`

Reply:
Ok, I used the same query as the one above because I imagined it in the same flow as the IO app.
But I agree with you that it is an intra service api so we can use the translational query